pkg/products/monitoringcommon/alertmanagerConfig.go

```diff
@@ -0,0 +1,266 @@
+package monitoringcommon
+
+import (
+	"context"
+	"fmt"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/resources"
+	l "github.com/integr8ly/integreatly-operator/pkg/resources/logger"
+	"github.com/integr8ly/integreatly-operator/pkg/resources/owner"
+	configv1 "github.com/openshift/api/config/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	v1 "github.com/openshift/api/route/v1"
+	"io/ioutil"
+	corev1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"os"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"strings"
+)
+
+const (
+	// alert manager configuration
+	alertManagerConfigSecretName            = "alertmanager-application-monitoring"
```

Comment:
Any reason these variables aren't stored in the config object?

Reply:
No reason they were copied over from https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/monitoring/reconciler.go#L52-L85

Reply:
let's move these to the config object then.

Reply:
Fixed